pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -551,6 +551,8 @@ func (plugin *kubenetNetworkPlugin) GetPodNetworkStatus(namespace string, name s
 	netnsPath, err := plugin.host.GetNetNS(id.ID)
 	if err != nil {
 		return nil, fmt.Errorf("Kubenet failed to retrieve network namespace path: %v", err)
+	} else if len(netnsPath) == 0 {
+		return nil, fmt.Errorf("Kubenet cannot retrieve pod IP with empty network namespace for %s/%s. This is usually because sandbox container has been torn down.", namespace, name)
```

Comment:
Only 2 options here: 
1. return an error (it will get logged)
2. not return an error ( I feel like GetPodNetworkStatus should do something at least if no network status is returned)

What if I add a specific error saying `failed to retrieve network status because pod no longer exists`? And add error handling at caller side